Kernel/Modules/AgentTicketCompose.pm

```diff
@@ -1264,60 +1375,316 @@ sub Run {
         # update Dynamic Fields Possible Values via AJAX
```

Comment:
This time above: We get the next states within the convergence loop now, below, so the above can be deleted.